backend/ttrs/urls.py

```diff
@@ -4,7 +4,9 @@
 from . import views
 
 urlpatterns = [
+    path('students/signin/', views.StudentSignIn.as_view()),
     path('students/', views.StudentList.as_view()),
+    path('students/signup/', views.StudentCreate.as_view()),
```

Comment:
Does 'students' need?
I think 'signup/' is also good.

Reply:
It would be better to use the *namespace* `student` because signup view will create a instance of `Student` model.